compose_rl/data/prompt_data.py

```diff
@@ -90,5 +98,19 @@ def __getitem__(self, idx: int) -> dict[str, Any]:
             prompt = prompt[:-truncate_len]
 
         prompt_len = torch.Tensor([len(prompt)]).to(dtype=torch.int64)
-
-        return {'prompt': prompt, 'prompt_len': prompt_len}
+        item_dict = {'prompt': prompt, 'prompt_len': prompt_len}
+
+        verified_answer = sample.get('verified_answer', None)
+        if verified_answer:
```

Comment:
```suggestion
        if verified_answer is not None:
```
in case verified answer is an empty string (`''`) for some reason
or should this case be caught as a separate case and dealt with separately?

Reply:
this is handled in the initial data processing script here: https://github.com/databricks/compose-rl/blob/abhay/data_loader_rlvr/scripts/data/unified_tokenize_dataset.py#L208

Reply:
so we will not have empty strings here